wp-web-push/wp-web-push.php

```diff
@@ -4,6 +4,8 @@
 Text Domain: wpwebpush
 */
 
+load_plugin_textdomain('wpwebpush', '', dirname(plugin_basename(__FILE__)) . '/lang');
```

Comment:
I don't know if it really matters, but we should load the textdomain only when strictly needed (we don't really need it in the public code). We can do it in a follow-up.